chapters/hpmor-chapter-015.tex

```diff
@@ -62,15 +71,15 @@ \chapter{Conscientiousness}
 Professor McGonagall stood up and moved over to the wall behind her desk, which held a polished wooden board. “There are many reasons why Transfiguration is dangerous, but one reason stands above all the rest.” She produced a short quill with a thick end and used it to sketch letters in red, which she then underlined, using the same marker, in blue:
 
 \begin{center}
-  \newsavebox{\hpbox}%
```

Comment:
Please can you restore the indentation here?

Reply:
oki, done